examples/packages/postgres-metabroker/images/Dockerfile.credential

```
@@ -0,0 +1,75 @@
+# Copyright 2020 SUSE
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM opensuse/leap:15.2 AS base_staging
+
+RUN zypper refresh
+RUN zypper --non-interactive install \
+    autoconf \
+    automake \
+    bison \
+    curl \
+    flex \
+    gcc \
+    git-core \
+    glibc-devel-static \
+    go1.14 \
```

```suggestion
    'golang(API) = 1.14' \
```
Or better, `'golang(API) >= 1.14'` or whatever it is we need.  There's no real reason to not using `go1.15`, I presume.

---

I dislike the non-deterministic `>=` for dependencies. I'll take the code suggestion, though.